src/Microsoft.VisualBasic/src/Microsoft/VisualBasic/Interaction.vb

```diff
@@ -1,10 +1,58 @@
 ' Licensed to the .NET Foundation under one or more agreements.
 ' The .NET Foundation licenses this file to you under the MIT license.
 ' See the LICENSE file in the project root for more information.
+Option Explicit On
+Option Strict On
 
 Namespace Microsoft.VisualBasic
 
-    Friend Module Interaction
+    Partial Public Module Interaction
+
+        Sub New()
+        End Sub
```

Comment:
Please remove the empty static constructor.

Reply:
Done

Reply:
This one I removed, some got added to allow the implementation to match the Ref file.